### PR TITLE
turn sel4bench results into web page

### PR DESCRIPTION
**Pull request by lsf37**
Turns json output from the sel4bench-hw action into a web page.

Example output is [here](https://htmlpreview.github.io/?https://raw.githubusercontent.com/seL4/website_pr_hosting/PR_153/localhost/About/Performance/index.html)

### CODE REVIEW

**Comment by lsf37** [comment]
Style is still complaining and I just saw that there are still two functions I wanted to factor out. Hang on a minute before reviewing.

**Comment by lsf37** [comment]
Ok, now ready for review.

File: sel4bench-web/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Content
+
+The entry point is the script [steps.sh].
+
+[Build] and [platform] configurations are in the respective yaml and python
+files in this repository. See these files for config documentation. They are
+shared with the [sel4bench-hw] and sel4bench actions.
+
+The main test driver is [build.py]. The [build.yml][Build] file contains the
```
Comment (Xaphiosis) [inline sel4bench-web/README.md:25]: `build.yml` or `builds.yml`?

File: sel4bench/build.py [hunk 2]
```diff
@@ -126,6 +133,203 @@ def make_runs(builds: List[Build]) -> List[Run]:
...
+        f.write('Time in cycles to invoke a user-level interrupt handler running in the same\n')
+        f.write('address space as the interrupted thread.</li>\n')
+
+        f.write('<li><strong>IPC call</strong>: ')
+        f.write('Time in cycles for invoking a server in a different address space on the same core.</li>\n')
+
+        f.write('<li><strong>IPC reply</strong>: ')
+        f.write('Time in cycles for a server replying to a client in a different address space on\n')
+        f.write('the same core.</li>\n')
+        f.write('</ul>')
```
Comment (Xaphiosis) [inline sel4bench/build.py:231]: could use triple-quote blocks to reduce the `f.write(` calls

**Review by Xaphiosis — APPROVED** [review]

File: sel4bench-web/README.md (hunk 1 above)
Comment (lsf37) [reply sel4bench-web/README.md:25]: Indeed `builds.yml`

File: sel4bench/build.py (hunk 2 above)
Comment (lsf37) [reply sel4bench/build.py:231]: The problem with the triple quotes is that the indent then looks ugly. We could use string concatenation to write a bit less, but I'm not sure it adds much.

**Pull request merged**